db/models.py

```diff
@@ -45,3 +50,72 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE)
```

Comment:
It is important to follow the company style guide. Please, choose one of the following formats and stick to it:
```python
# Break before the opening and after the closing parentheses.
  user = models.ForeignKey(
        to=settings.AUTH_USER_MODEL,
        on_delete=models.CASCADE
)

# No breaks after or before parentheses
  user = models.ForeignKey(to=settings.AUTH_USER_MODEL,
                           on_delete=models.CASCADE)
```

Reply:
Thank you for your review! I've chosen the first variant of parentheses style and replaced get method with movie_session_id argument.